src/pages/404/PageNotFound.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import "./pageNotFound.css"
+import pageNotFound from "../../assets/images/404.svg"
+export function PageNotFound() {
+  return (
+    <div className='show-404'>
+        <h1>You took a wrong turn</h1>
+        <img className="show-not-found responsive-img" src={pageNotFound} alt="page not found"></img>
+    </div>
+  )
+}
```

Comment:
appreciation - it's good that you have implemented the 404 page

Reply:
Thanks @anik31